### PR TITLE
FAQ: So many machine thingys

### PR DESCRIPTION
**Pull request by cgwalters**
Describe our relationship with machine API a bit.

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -23,3 +23,17 @@ Ignition JSON directly.
 Today, the MCO only blocks on upgrades of control plane nodes.  `oc get clusterversion` effectively reports the version of the control plane. 
 
 To watch rollout of worker nodes, you should look at `oc describe machineconfigpool/worker` (as well as other custom pools, if any).
+
+## Q: How does this relate to Machine API?
+
+Confusingly, there are two fundamental operators in OpenShift 4 that both include "machine" in their name:
```
Comment (kikisdeliveryservice) [inline docs/FAQ.md]: Strike this para.

Comment (kikisdeliveryservice) [reply docs/FAQ.md]: or at least "confusingly"

File: docs/FAQ.md [hunk 2]
```diff
@@ -23,3 +23,17 @@ Ignition JSON directly.
...
+
+This repository (Machine Config Operator) manages "inside" the OS (and targets specifcally RHCOS).
```
Comment (kikisdeliveryservice) [inline docs/FAQ.md]: The Machine Config Operator manages...

File: docs/FAQ.md (hunk 1 above)
Comment (ashcrow) [reply docs/FAQ.md]: Or maybe "It may seem confusing but there are two..."

**Comment by ashcrow** [comment]
>  FAQ: So many machine thingys

:rofl: :+1:

**Comment by cgwalters** [comment]
Tweaked.

**Comment by kikisdeliveryservice** [comment]
TBH really torn on whether this should be `thingys` or `thingies`. The latter seems grammatically correct but the former _feels_ correct. :laughing:

**Pull request merged**